tests/covalent_tests/workflow/electron_test.py

```diff
@@ -643,3 +643,22 @@ def workflow(x):
     assert (
         workflow.transport_graph.get_node_value(0, "status") == RESULT_STATUS.PENDING_REPLACEMENT
     )
+
+
+def test_electron_pow_method():
+    electron = Electron(function=None)
+    assert hasattr(electron, "__pow__")
+
+
+def test_workflow():
+    @ct.electron
+    def g(x):
+        return 42 * x
+
+    @ct.lattice
+    def workflow(x):
+        res = g(x)
+        return res**2
+
+    result = workflow(2)
+    assert result == 7056  # Expected result for (42 * 2) ** 2
```

Comment:
Thanks for your comment! 
Could you give me advice?- I'm trying to specify mocker.patch("covalent._workflow.electron.Electron.get_op_function") in the test, but due to the fact that the electron function is imported into _workflow in init, confusion occurs and mocker perceives covalent._workflow.electron as a function:
ModuleNotFoundError: No module named 'covalent._workflow.electron.Electron'; 'covalent._workflow.electron' is not a package

Reply:
I sent current (not final) version of pow method test, wait for advice on how to fix it

Reply:
>I also tried mocker.path, as in test_lattice_draw, but due to the import of decorator electron in __init__, there are some problems with importing inside mocker.patch methods covalent._workflow.electron.Electron.__pow__ or .get_op_function, for mocker.patch electron is a decorator, not a script

Yeah, I can see why that could be an issue here, but I think you have the right approach to instead mock the object's function directly.

>but get AssertionError: Expected 'get_op_function' to have been called once. Called 0 times.

Ah, that's actually expected since we've mocked the `__pow__` function. It was my bad, didn't consider that. So, what you wanna do is instead only mock the `get_op_function` and assert that it has been called with the right arguments including `**` using the `assert_called_with` instead of `assert_called_once` and that should fix things.